src/components/Provider/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { Grommet } from 'grommet';
-import { PartialObject, cloneDeep } from 'lodash/index';
```

Comment:
I guess this was b/c of an IDE auto-import.
Also dropped `PartialObject` in favor of the TS 2.1 `Partial`, which we already use in the `Map` component.